### DISCUSSION TITLE
Incomplete Category and Subcategory information from Catalog Item API

### DISCUSSION THREAD

**Discussion by ANarayanan-Cervello** [amzn/selling-partner-api-models#3254] [General]
I am using the Catalog Items API endpoint to retrieve product attribute
information about ASINS.

However ,for the attributes 'product_subcategory' and 'product_category' we are getting response
in the below format :

```
'product_subcategory': [{'value': '12102010',
'marketplace_id': 'ATVPDKIKX0DER'}]

'product_category': [{'value': '12102000',
'marketplace_id': 'ATVPDKIKX0DER'}]

```
However, on the portal ,the values for Category and SubCategory are in the format :

SubCategory: 2010 Formula
Category : 2000 Baby Care

ASIN used here is : B000BIVY56

Why is there a difference between the Category/Subcategory attribute value displayed on the portal versus
the values coming as response from the API?

MarketplaceID: 'ATVPDKIKXODER' or 'US'
endpoint : `'/catalog/2020-12-01/items/{ASIN}', method='GET'`

**Comment by chapmanjw** [comment]
The value in the `attributes` data is the actual value for those attributes (the numeric values). Additionally, the `vendorDetails` section contains the numeric values in the v2020-12-01 version. We are expanding the `vendorDetails` to include the display names for these. See https://github.com/amzn/selling-partner-api-docs/discussions/2017 for a preview of those changes.

Additionally, the JSON Schema for the product types in the Product Type Definitions API includes both the values and display names of the categories and subcategories for a given product type.

**Reply by ANarayanan-Cervello** [reply]
I did go through the Product Type Definitions API ,but in the absence of productType , I don't seem to be able to use it. Is there a way to obtain the product type after just passing an SKU?
Even with productType,I don't see any piece of information that reveals the particular category / subcategory to which a SKU belongs.
@chapmanjw  Would it be possible for you to demonstrate an example as to how I could use this, in case my understanding is wrong?

Meanwhile, how long would it take to add _vendorDetails_ onto the CatalogItems API response?
